runtime.md

```diff
@@ -33,24 +33,79 @@ This is provided so that consumers can find the container's configuration and ro
 }
 ```
 
-## Lifecycle
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
+
+OPEN: we should consider specifying the options/args for each operation.
+For example, checkpoint (for runc) seems to have a lot of params.
 
 ### Create
 
-Creates the container: file system, namespaces, cgroups, capabilities.
+Using the `config.json` and `runtime.json` files, along with the file system path referenced in the `config.json`, this operation MUST create a new container.
+This includes creating the relevant namespaces, cgroups and configuring the appropriate capabilities for the container.
+
+### Start
+
+This operation MUST start a previously created container.
+Starting a container MUST create a new process, as specified by the `config.json` file within the scope of the container.
+Attempting to start an already running container MUST have no effect on the container and MUST generate an error.
+Starting a stopped container, one that had been previously be running, MUST create a new process, as specified by the `config.json` file, within the scope of the container.
+
+OPEN: the entire "generate an error" stuff in here is something we should discuss.
```

Comment:
On Mon, Jan 04, 2016 at 10:49:48AM -0800, Doug Davis wrote:
> > +OPEN: the entire "generate an error" stuff in here is something we should discuss.
> 
> I purposely used the word "generate" instead of "report" or
> something else that implies the end user sees it. I think that gets
> into implementation details. For example, some impl might just log
> everything and their "user" never sees any of it and I think that's
> valid from this specs perspective.  The point is that we all agree
> its an error condition and the action did not happen but how the
> impl deal with the error is up to it.

That sounds like a useful distinction to me, and I think its worth
spelling it in this PR.  How about adding an error-handling section
somewhere (the glossary?  #107) that translates your generate
vs. report disctinction into spec language, and then just use “MUST
generate an error” like you already do in this section.

Once we do that, we can drop the OPEN issue here, and open a mailing
list thread if/when we want something in the spec about the reporting
aspect.